kythe/go/extractors/cmd/gotool/gotool.go

```diff
@@ -50,6 +50,20 @@ var (
 	canonicalizePackageCorpus = flag.Bool("canonicalize_package_corpus", false, "Whether to use a package's canonical repository root URL as their corpus")
 )
 
+// Wrapper type for using flag.Var() to collect a list of strings.
+type arrayFlags struct {
+	array *[]string
+}
+
+func (af *arrayFlags) String() string {
+	return strings.Join(*af.array, ",")
+}
+
+func (af *arrayFlags) Set(value string) error {
+	*af.array = append(*af.array, value)
+	return nil
+}
+
```

Comment:
I think, based on my recent, change, you'll also need a Get implementation.

Also, maybe "stringArrayFlag" as the type name?